org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/PublisherGenerator.java

```diff
@@ -539,6 +540,20 @@ public void generate() throws Exception {
     }
   }
 
+  private byte[] fixLang(byte[] b, String l) {
```

Comment:
Might be better to name this fillLangTemplate, unless this is actually fixing some incorrect content.

Reply:
ok